src/components/Navbar.js

```diff
@@ -0,0 +1,38 @@
+import React, { useState, useEffect } from 'react';
+import { NavLink } from 'react-router-dom';
+import { library } from '@fortawesome/fontawesome-svg-core';
+import { faAngleLeft } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+
+library.add(faAngleLeft);
+
+const Navbar = () => {
+  const [activeIndex, setActiveIndex] = useState(0);
+
+  useEffect(() => {
+    if (window.location.pathname === '/missions') {
+      setActiveIndex(1);
+    } else if (window.location.pathname === '/profile') {
+      setActiveIndex(2);
+    }
+  }, []);
+
+  const handleClick = (index) => {
+    setActiveIndex(index);
+  };
+
+  return (
+    <header className="nav-header">
+      <nav>
+        <ul>
+          <NavLink to="/" className={`routing ${activeIndex === 0 ? 'activated' : ''}`} onClick={(e) => handleClick(0, e)}>
+            <FontAwesomeIcon className="font-icon" icon="angle-left" size="1.5x" color="#fff" />
+          </NavLink>
+        </ul>
+      </nav>
+    </header>
+
```

Comment:
- [ ] As it is the case on the original design, Kinldy make sure you have the navigation bar set on your home page as is the case on the original design.
### Original design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/233449487-65ef116b-0456-484e-b913-0e3d4f00b8a6.png)

### Your design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/233449639-d3804e78-f6fe-4b04-a67f-7294575d8682.png)